src/metabase/driver/h2.clj

```diff
@@ -168,68 +168,69 @@
-    :VARCHAR_CASESENSITIVE       :type/Text
-    :VARCHAR_IGNORECASE          :type/Text
-    :YEAR                        :type/Integer
-    (keyword "DOUBLE PRECISION") :type/Float} database-type))
+  ({:ARRAY                               :type/*
+    :BIGINT                              :type/BigInteger
+    :BINARY                              :type/*
+    :BIT                                 :type/Boolean
+    :BLOB                                :type/*
+    :BOOL                                :type/Boolean
+    :BOOLEAN                             :type/Boolean
+    :BYTEA                               :type/*
+    :CHAR                                :type/Text
+    :CHARACTER                           :type/Text
+    :CLOB                                :type/Text
+    :DATE                                :type/Date
+    :DATETIME                            :type/DateTime
+    :DEC                                 :type/Decimal
+    :DECIMAL                             :type/Decimal
+    :DOUBLE                              :type/Float
+    :FLOAT                               :type/Float
+    :FLOAT4                              :type/Float
+    :FLOAT8                              :type/Float
+    :GEOMETRY                            :type/*
+    :IDENTITY                            :type/Integer
+    :IMAGE                               :type/*
+    :INT                                 :type/Integer
+    :INT2                                :type/Integer
+    :INT4                                :type/Integer
+    :INT8                                :type/BigInteger
+    :INTEGER                             :type/Integer
+    :LONGBLOB                            :type/*
+    :LONGTEXT                            :type/Text
+    :LONGVARBINARY                       :type/*
+    :LONGVARCHAR                         :type/Text
+    :MEDIUMBLOB                          :type/*
+    :MEDIUMINT                           :type/Integer
+    :MEDIUMTEXT                          :type/Text
+    :NCHAR                               :type/Text
+    :NCLOB                               :type/Text
+    :NTEXT                               :type/Text
+    :NUMBER                              :type/Decimal
+    :NUMERIC                             :type/Decimal
+    :NVARCHAR                            :type/Text
+    :NVARCHAR2                           :type/Text
+    :OID                                 :type/*
+    :OTHER                               :type/*
+    :RAW                                 :type/*
+    :REAL                                :type/Float
+    :SIGNED                              :type/Integer
+    :SMALLDATETIME                       :type/DateTime
+    :SMALLINT                            :type/Integer
+    :TEXT                                :type/Text
+    :TIME                                :type/Time
+    :TIMESTAMP                           :type/DateTime
+    (keyword "TIMESTAMP WITH TIME ZONE") :type/DateTime
```

Comment:
@camsaul I added this mapping. Is it correct or should it be `:type/DateTimeWithTz`?

Reply:
It should be `:type/DateTimeWithTZ` (capital `Z`)